src/binding.ts

```diff
@@ -47,6 +48,38 @@ export class BindingService {
     return BindingService._instance;
   }
 
+  async editBinding(workspaceFolderItem: WorkspaceFolder): Promise<void> {
```

Comment:
This is really minor but I think type name `WorkspaceFolder` is really confusing with `VSCode.WorkspaceFolder`. I think `WorkspaceFolderItem` could be clearer? Because it denotes actually type of tree view item for WS folder